_data/4cc.yaml

```diff
@@ -40,19 +40,19 @@
   usage:
   - APTicket
 - code: CSEC
-  description: Chip security mode
+  description: Chip Security Mode
   usage:
   - APTicket
 - code: ECID
   description: Exclusive Chip ID
   usage:
   - APTicket
 - code: EPRO
-  description:
+  description: Effective Production Status
```

Comment:
Is this production as in demotion?

Reply:
yes, like CPRO

Reply:
If I understand CPRO would be 1 (true) and EPRO would be 0 (false) if it’s a prod device demoted?